pkgs/applications/audio/ardour/default.nix

```diff
@@ -56,20 +58,39 @@
 }:
 stdenv.mkDerivation rec {
   pname = "ardour";
-  version = "6.9";
+  version = "7.0";
 
-  # don't fetch releases from the GitHub mirror, they are broken
+  # We can't use `fetchFromGitea` here, as attempting to fetch release archives from git.ardour.org
+  # result in an empty archive. See https://tracker.ardour.org/view.php?id=7328 for more info.
   src = fetchgit {
     url = "git://git.ardour.org/ardour/ardour.git";
     rev = version;
-    sha256 = "0vlcbd70y0an881zv87kc3akmaiz4w7whsy3yaiiqqjww35jg1mm";
+    hash = "sha256-CCwkXRNiVnrlxUn3QEt2e/w9aGdzZJ306O9/1ERj7/c=";
+  };
+
+  bundledContent = fetchzip {
+    url = "https://web.archive.org/web/20221026200824/http://stuff.ardour.org/loops/ArdourBundledMedia.zip";
+    hash = "sha256-IbPQWFeyMuvCoghFl1ZwZNNcSvLNsH84rGArXnw+t7A=";
+    # archive does not contain a single folder at the root
+    stripRoot = false;
   };
```

Comment:
@magnetophon I'm having some trouble getting the bundled content installed. The hierarchy of the zip looks like:

```
$ ls ArdourBundledMedia
'MIDI Beats'  'MIDI Chords'  'MIDI Progressions'
```

The download completes successfully, but upon attempting to `install` the content later on, I get the following error:

```
install: creating directory '/nix/store/v2pbqk0yl7c51mnigvfjnxpspr16rky6-ardour-7.0/share/media'
install: omitting directory '/nix/store/vzmx96hjs7k1czfb0c8akpfzj8wbxac6-source'
error: builder for '/nix/store/bixsrrnilbcnx69lj6p8aks0w33pwjzd-ardour-7.0.drv' failed with exit code 1;
       last 10 log lines:
       > removed '/nix/store/v2pbqk0yl7c51mnigvfjnxpspr16rky6-ardour-7.0/share/icons/hicolor/256x256/apps/ardour7.png'
       > 'gtk2_ardour/resources/Ardour-icon_256px.png' -> '/nix/store/v2pbqk0yl7c51mnigvfjnxpspr16rky6-ardour-7.0/share/icons/hicolor/256x256/apps/ardour7.png'
       > removed '/nix/store/v2pbqk0yl7c51mnigvfjnxpspr16rky6-ardour-7.0/share/icons/hicolor/512x512/apps/ardour7.png'
       > 'gtk2_ardour/resources/Ardour-icon_512px.png' -> '/nix/store/v2pbqk0yl7c51mnigvfjnxpspr16rky6-ardour-7.0/share/icons/hicolor/512x512/apps/ardour7.png'
       > install: creating directory '/nix/store/v2pbqk0yl7c51mnigvfjnxpspr16rky6-ardour-7.0/share/man'
       > install: creating directory '/nix/store/v2pbqk0yl7c51mnigvfjnxpspr16rky6-ardour-7.0/share/man/man1'
       > 'ardour.1' -> '/nix/store/v2pbqk0yl7c51mnigvfjnxpspr16rky6-ardour-7.0/share/man/man1/ardour.1'
       > 'MIDI Beats'  'MIDI Chords'  'MIDI Progressions'
       > install: creating directory '/nix/store/v2pbqk0yl7c51mnigvfjnxpspr16rky6-ardour-7.0/share/media'
       > install: omitting directory '/nix/store/vzmx96hjs7k1czfb0c8akpfzj8wbxac6-source'
       For full logs, run 'nix log /nix/store/bixsrrnilbcnx69lj6p8aks0w33pwjzd-ardour-7.0.drv'.
```

Questions:

1. Should I be installing this to `$out/share/media`? That's my best guess based on [the original install script](https://github.com/Ardour/ardour/blob/14da117bc880ebcc8a805348a97acdae31b2753b/tools/linux_packaging/build#L758), but I'm not entirely sure.
2. Am I invoking `install` correctly? I assume not since I'm getting an error code.

Still a noob at nix packaging, so some advice would be appreciated :)

Reply:
@anoadragon453 Could you post (a link to) the full error?
Run ``nix log /nix/store/bixsrrnilbcnx69lj6p8aks0w33pwjzd-ardour-7.0.drv``.

Reply:
Sure, you can see it at https://gist.github.com/anoadragon453/b4cafdeb3e32e926c8c3375394b79e1e.

Reply:
Do you see anything immediately wrong? If not, I'd be tempted to de-scope this addition and instead add the bundled content in a separate PR, if only to allow people to start playing with Ardour 7.x and test it out.

Reply:
Have figured it out, it needs to be put in `/share/ardour7/media` :)

Commits incoming (after I check it builds locally successfully).

Reply:
Done in https://github.com/NixOS/nixpkgs/pull/196290/commits/67929885e8099106d8a5185c5bf1bde5b457fc10.